src/Commands/GitPackage.php

```diff
@@ -82,21 +82,25 @@ public function handle()
         $this->conveyor->checkIfPackageExists();
         $this->makeProgress();
 
-        // Create the package directory
```

Comment:
Are you sure this section has to be removed completely?

Reply:
Will need to test a few more times, however my git clone worked perfectly without a `packages` folder when I tested. But to be fair, there is no harm in keeping it there. May just need to be deleted if the `git clone` fails. What do you think?

Reply:
I prefer to keep it in there.

Reply:
Ok, will write the functionality back in shortly.